tools/logger/convert.c

```diff
@@ -472,7 +472,7 @@ static int fetch_entry(const struct convert_config *config,
 			}
 			if (ret != size)
 				log_err(config->out_fd,
-					"Partial read of %u bytes of %lu.\n",
+					"error: Partial read of %u bytes of %lu.\n",
 					ret, size);
```

Comment:
I remember other error message prefix is `Error`, could you check above comment to warp the Error prefix in the `log_err`?

Reply:
In this file there are both versions in usage, `Error` and `error`

Reply:
in `logger.c` `error: ` version is used consistently, so I stick to this.